### DISCUSSION TITLE
Made three repo's today but can only push to one

### DISCUSSION THREAD

**Discussion by BataBoom** [community/community#86754] [New to GitHub]
Made three repo's today but can only push to one. The other two I get this error: 

ERROR: Permission to BataBoom/OMCC.git denied to deploy key
fatal: Could not read from remote repository.
Authenticated to github.com ([192.30.255.113]:22).
debug1: channel 0: new [client-session]
debug1: Entering interactive session.
debug1: pledge: network
debug1: client_input_global_request: rtype hostkeys-00@openssh.com want_reply 0
debug1: Sending environment.
debug1: Sending env LANG = en_US.UTF-8
debug1: Sending command: git-receive-pack 'BataBoom/OMCC.git'
debug1: client_input_channel_req: channel 0 rtype exit-status reply 0
ERROR: Permission to BataBoom/OMCC.git denied to deploy key
debug1: channel 0: free: client-session, nchannels 1
debug1: fd 0 clearing O_NONBLOCK
Transferred: sent 2776, received 2516 bytes, in 0.2 seconds
Bytes per second: sent 12020.8, received 10894.9
debug1: Exit status 1
fatal: Could not read from remote repository.

Odd says "Could not read from remote repository." I can pull, just not push. 

Configs are almost identical with the exception of remote url's. The git is initialized in all the repos. I'm using the same Machine with the same SSH key. Ive been trying to figure this out for far too long (couple hrs). Ive made 33 commits to the repo I do have access to and for that reason you should help me out, im losing it! https://github.com/BataBoom/SatoshiCast

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 are you still experiencing this?

**Reply by BataBoom** [reply]
yes 


ssh -v -o "IdentitiesOnly=yes" -i ~/.ssh/id_rsa git@github.com
ssh -T -ai ~/.ssh/id_rsa git@github.com
Enter passphrase for key '/home/bataboom/.ssh/id_rsa': 
Hi BataBoom! You've successfully authenticated, but GitHub does not provide shell access.
bataboom@static:$ git remote add survivor git@github.com:BataBoom/Survivor.git
bataboom@static:/var/www/survivor/packages/package-skeleton-laravel$ git push -u survivor main
ERROR: Permission to BataBoom/Survivor.git denied to deploy key
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
??

**Comment by BataBoom** [comment]
bataboom@static:/var/www/merchant$ ssh -T merchant
Hi BataBoom/SatsBB! You've successfully authenticated, but GitHub does not provide shell access.
bataboom@static:$ /var/www/merchant$ git push -u origin main
ERROR: Permission to BataBoom/SatsBB.git denied to deploy key
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Ive added this brand new rsa pubkey to Deploy Keys. I had it in account ssh keys.. but it would say key already in use. So deleted it from acct settings. https://github.com/BataBoom/SatsBB key has access to write.

[core]
        repositoryformatversion = 0
        filemode = true
        bare = false
        logallrefupdates = true
[remote "origin"]
        url = git@github.com:BataBoom/SatsBB.git
        fetch = +refs/heads/*:refs/remotes/origin/*

I am using a dedicated server, but I dont see the problem. is cause my acct is free? I seen something about 500MB cap, i dont believe ive passed that limit for all repos combined. is there a way to check?

**Comment by DarrenChochian** [comment]


**Reply by BataBoom** [reply]
Hi thanks for the reply. 

Prior to reading I logged into an old VPS, and was able to push perfectly fine. I figured there must be some git global config, though reading this helped more than playing with that. I was adding so many keys/etc that I edited the SSH config file. I would name the Host's just words, but it appears I made the host on one entry github.com .. and it was using a specific repository deploy key. After commenting this out, and running that command I was prompted to hit the password from my authorized_keys file. 😳 Now everything is wonderful again.

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
